### ISSUE TITLE
Frozen training

### ISSUE THREAD

**Issue by oraculseed** [Rudrabha/Wav2Lip#173]
Hi. After starting the training, nothing happens. 
![w2l1](https://user-images.githubusercontent.com/25270616/103178968-92335a80-4898-11eb-9be6-547ab993a8d2.PNG)
But the memory of the video card goes a little. 
![w2l2](https://user-images.githubusercontent.com/25270616/103178970-95c6e180-4898-11eb-99ed-6dee7b32bee9.PNG)
Any ideas? Thx!

**Comment by oraculseed** [comment]
On CPU frozen too.

**Comment by prajwalkr** [comment]
The dataloader is stuck in an infinite loop as it is unable to find the files. You can see that the dataloader has "continue" statements if there are errors in particular samples. Ensure that the dataloader is able to successfully load samples.

**Comment by oraculseed** [comment]
Thx! It's solved! But there was a new problem. 
![w2l3](https://user-images.githubusercontent.com/25270616/103194853-eae31180-48f1-11eb-95ce-bd4454f31b1a.PNG)
How can this be solved ?

**Comment by prajwalkr** [comment]
There is an issue with your audio input. It is of the wrong shape.

**Comment by oraculseed** [comment]
> There is an issue with your audio input. It is of the wrong shape.

Why ? The audio data matches the required ones. 
![w2l4](https://user-images.githubusercontent.com/25270616/103208075-d6624180-4910-11eb-9844-71e07e4ce24c.PNG)
What do you need to look at to make sure that they are incorrect ?

**Comment by prajwalkr** [comment]
You need to use a batch size > 1 I think for batchnorm? Can you try with batch size > 1?

**Comment by oraculseed** [comment]
> You need to use a batch size > 1 I think for batchnorm? Can you try with batch size > 1?

My params:

```
hparams = HParams(
	num_mels=80,  # Number of mel-spectrogram channels and local conditioning dimensionality
	#  network
	rescale=True,  # Whether to rescale audio prior to preprocessing
	rescaling_max=0.9,  # Rescaling value
	
	# Use LWS (https://github.com/Jonathan-LeRoux/lws) for STFT and phase reconstruction
	# It"s preferred to set True to use with https://github.com/r9y9/wavenet_vocoder
	# Does not work if n_ffit is not multiple of hop_size!!
	use_lws=False,
	
	n_fft=800,  # Extra window size is filled with 0 paddings to match this parameter
	hop_size=200,  # For 16000Hz, 200 = 12.5 ms (0.0125 * sample_rate)
	win_size=800,  # For 16000Hz, 800 = 50 ms (If None, win_size = n_fft) (0.05 * sample_rate)
	sample_rate=16000,  # 16000Hz (corresponding to librispeech) (sox --i <filename>)
	
	frame_shift_ms=None,  # Can replace hop_size parameter. (Recommended: 12.5)
	
	# Mel and Linear spectrograms normalization/scaling and clipping
	signal_normalization=True,
	# Whether to normalize mel spectrograms to some predefined range (following below parameters)
	allow_clipping_in_normalization=True,  # Only relevant if mel_normalization = True
	symmetric_mels=True,
	# Whether to scale the data to be symmetric around 0. (Also multiplies the output range by 2, 
	# faster and cleaner convergence)
	max_abs_value=4.,
	# max absolute value of data. If symmetric, data will be [-max, max] else [0, max] (Must not 
	# be too big to avoid gradient explosion, 
	# not too small for fast convergence)
	# Contribution by @begeekmyfriend
	# Spectrogram Pre-Emphasis (Lfilter: Reduce spectrogram noise and helps model certitude 
	# levels. Also allows for better G&L phase reconstruction)
	preemphasize=True,  # whether to apply filter
	preemphasis=0.97,  # filter coefficient.
	
	# Limits
	min_level_db=-100,
	ref_level_db=20,
	fmin=55,
	# Set this to 55 if your speaker is male! if female, 95 should help taking off noise. (To 
	# test depending on dataset. Pitch info: male~[65, 260], female~[100, 525])
	fmax=7600,  # To be increased/reduced depending on data.

	###################### Our training parameters #################################
	img_size=512,
	fps=25,
	
	batch_size=8,
	initial_learning_rate=1e-4,
	nepochs=200000000000000000,  ### ctrl + c, stop whenever eval loss is consistently greater than train loss for ~10 epochs
	num_workers=16,
	checkpoint_interval=3000,
	eval_interval=3000,
    save_optimizer_state=True,

    syncnet_wt=0.0, # is initially zero, will be set automatically to 0.03 later. Leads to faster convergence. 
	syncnet_batch_size=64,
	syncnet_lr=1e-4,
	syncnet_eval_interval=10000,
	syncnet_checkpoint_interval=10000,

	disc_wt=0.07,
	disc_initial_learning_rate=1e-4,
)
```
batch_size = 8 , syncnet_batch_size=64

**Comment by prajwalkr** [comment]
> ![w2l4](https://user-images.githubusercontent.com/25270616/103208075-d6624180-4910-11eb-9844-71e07e4ce24c.PNG)

The first dimension in the tensor is batch size. As you can clearly see, it is 1.

**Comment by oraculseed** [comment]
Thank you. Great. Added more data and it worked. But now here is such a mistake.
![w2l5](https://user-images.githubusercontent.com/25270616/103235341-f4e92c80-4952-11eb-92dd-404ff975f739.PNG)
Looks like I'm doing something wrong.

**Comment by oraculseed** [comment]
Solved!

**Comment by wowopc** [comment]
> 在CPU上也冻结了。

https://github.com/Rudrabha/Wav2Lip/issues/173#issuecomment-751512641

**Comment by wowopc** [comment]
Hello, I have the same problem as you. Can you tell me how to solve it

**Comment by wowopc** [comment]
Frozen training

**Comment by wowopc** [comment]
@oraculseed How can I contact you

**Comment by ayush714** [comment]
> Thx! It's solved! But there was a new problem. ![w2l3](https://user-images.githubusercontent.com/25270616/103194853-eae31180-48f1-11eb-95ce-bd4454f31b1a.PNG) How can this be solved ?

Hi @oraculseed  Can you please tell me how you solved it?
